internal/pkg/crawl/utils.go

```diff
@@ -52,8 +53,10 @@ func (c *Crawl) checkIncludedHosts(host string) bool {
 
 func (c *Crawl) handleCrawlPause() {
 	for {
-		if float64(utils.GetFreeDiskSpace(c.JobPath).Avail)/float64(GB) <= 20 {
-			logrus.Errorln("Not enough disk space. Please free some space and restart the crawler.")
+		spaceLeft := float64(utils.GetFreeDiskSpace(c.JobPath).Avail) / float64(GB)
+		if spaceLeft <= float64(c.MinSpaceRequired) {
+			logrus.Errorln(fmt.Sprintf("Not enough disk space: %d GB required, %f GB available", c.MinSpaceRequired, spaceLeft))
+			logrus.Errorln("Please free some space and restart the crawler.")
```

Comment:
Only freeing some space is required, the crawler will unpause itself when it has enough disk space.